docs/quickstart/apollo_6_0_quick_start.md

```diff
@@ -0,0 +1,84 @@
+# Apollo 6.0 Quick Start Guide
+
+The following guide serves as a user manual for launching the Apollo upgraded software and hardware stack on vehicle, which
+is similar to Apollo 5.5.
+
+The latest Apollo upgrade, Quick Start Guide focuses on the new features, like software and third party dependencies update. For general Apollo concepts, please refer to
+[Apollo 3.5 Quick Start](https://github.com/ApolloAuto/apollo/blob/master/docs/quickstart/apollo_3_5_quick_start.md)
+
+## Contents
+
+- [Calibration Guide](#calibration-guide)
+- [Hardware and Software Installation](#hardware-and-software-installation)
+- [Dreamview Usage Table](#dreamview-usage-table)
+- [Onboard Test](#onboard-test)
+
+## Calibration Guide
+
+Apollo currently offers a robust calibration service to support your calibration requirements from LiDARs to IMU to Cameras. This service is currently being offered to select partners only as part of our Data Pipeline service offering. If you would like to learn more about the calibration service, please reach out to us via email: **apollopartner@baidu.com**
+
```

Comment:
1) Two `currently` seems repetitive. The first one can be removed.
2) select partners -> selected partners